mRemoteV1/packages.config

```diff
@@ -8,7 +8,9 @@
     <package id="DockPanelSuite.ThemeVS2013" version="3.0.6" targetFramework="net46" />
     <package id="DockPanelSuite.ThemeVS2015" version="3.0.6" targetFramework="net46" />
     <package id="Geckofx45" version="45.0.32" targetFramework="net46" />
+    <package id="Google.Protobuf" version="3.5.1" targetFramework="net46" />

```

Comment:
If the lib is not in use, remove the nuget as well per my previous comment.

Reply:
Ahh.. I think this is a hangover of looking into Entity Foundation as suggested by @sparerd in #1177 - I suspect it's hanging around from that. (same as .csproj change too) Will fix.

Reply:
Done - pushed a rebase to include this in the main commitDone - pushed a rebase to include this in the main commit